src/test/java/com/example/PascalTriangleTest.java

```diff
@@ -0,0 +1,26 @@
+package com.example;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import static com.example.PascalTriangle.*;
+
+public class PascalTriangleTest {
+    @Test
+    public void testNthRow() {
+        int n = 0;
```

Comment:
Do you need the `n` variable at all?
You can just call: ` assertArrayEquals(new int[]{1}, NthRow(1));`

Reply:
OK. Willing to push a new version soon.